seagull.mk

```diff
@@ -0,0 +1,27 @@
+# Copyright 2014 The Android Open Source Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+$(call inherit-product, device/sony/seagull/aosp_d5103.mk)
```

Comment:
I think we need to change one line from "call inherit-product" to "call inherit-product-if-exists" withi, aosp_d5103.mk
If you don't do this, then the make dependency will fail when you don't have the blobs and thus the downloading script will not be triggered